src/etc/inc/pkg-utils.inc

```diff
@@ -880,6 +880,13 @@ function delete_package_xml($package_name, $when = "post-deinstall") {
 	$static_output .= gettext("done.") . "\n";
 	update_output_window($static_output);
 	write_config("Removed {$package_name} package.\n");
+
+	/* remove package entry from /etc/syslog.conf if needed */
+	/* this must to be done after removing the entries from config.xml */
```

Comment:
I couldn't decide between *must* and *needs to* so I made a compromise :stuck_out_tongue_winking_eye: